development/gcp/cloud-run/scan-logs-for-secrets/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM golang:1.19.3-alpine3.16 as builder
```

Comment:
Could you bump this image or add this to the dependabot?

Reply:
Updated to 1.19.5-alpine3.17